### DISCUSSION TITLE
What are some simplified websites were I could learn more JavaScript and the basics of SQL and PHP?

### DISCUSSION THREAD

**Discussion by emullallphotography** [community/community#60795] [Programming Help]
### Body

Thanks for the help!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by nirav-gajera** [comment]
- http://www.codecademy.com/
- http://www.w3schools.com/
- http://www.freecodecamp.org/
- MDN Web Docs - developer.mozilla.org (https://developer.mozilla.org/en-US/docs/Glossary/PHP)
- http://www.sololearn.com/

-These websites offer a combination of tutorials, interactive exercises, and practical examples to help you grasp the fundamentals of JavaScript, SQL, and PHP.

**Comment by Beargruug** [comment]
Hey, 

i can recommend to watch a course here at udemy (https://www.udemy.com/course/learn-sql-and-php-to-build-powerful-and-secure-web-apps/?kw=javascript+sql+php&src=sac)

Maybe also another course would be good! 

Hope you'll like it!
